Frameworks/CoreGraphics/CGGraphicBufferImage.mm

```diff
@@ -82,12 +223,16 @@
     _nativeTexture = nativeTexture;
     _nativeTextureLocking = locking;
     _nativeTextureLocking->RetainDisplayTexture(_nativeTexture);
+    _renderTarget = nullptr;
 }
 
 CGGraphicBufferImageBacking::~CGGraphicBufferImageBacking() {
     EbrDecrement((volatile int*)&imgDataCount);
     TraceVerbose(TAG, L"Destroyed (freeing fasttexture 0x%x) - Number of images: %d", _nativeTexture, imgDataCount);
 
+    if (_renderTarget != nullptr) {
+        _renderTarget->Release();
+    }
```

Comment:
Ah, it is not. hm.

---
In reply to: [80768683](https://github.com/Microsoft/WinObjC/pull/1040#discussion_r80768683) [](ancestors = 80768683)